tests/testgraphelement.py

```diff
@@ -130,13 +133,15 @@ def test_graph_redim_nodes(self):
 class TriMeshTests(ComparisonTestCase):
 
     def setUp(self):
-        self.nodes = [(0, 0, 0), (0.5, 1, 1), (1., 0, 2), (1.5, 1, 3)]
+        self.nodes = [(0, 0, 0), (0.5, 1, 1), (1., 0, 2), (1.5, 1, 4)]
```

Comment:
The last value wasn't tested before?

Reply:
It was, but it wasn't a very good test. I've adjusted the test so it properly tests how the Graph index dimension is handled.